src/api/lib/tasks/dev.rake

```diff
@@ -416,8 +416,9 @@ def request_for_staging(staging_project, maintainer_project, suffix)
 end
 
 def create_and_assign_project(project_name, user)
-  create(:project, name: project_name)
-  create(:relationship, project: project_name, user: user, role: Role.hashed['maintainer'])
+  project = create(:project, name: project_name)
```

Comment:
This isn't a blocker, but it could be rewritten with [tap](https://ruby-doc.org/core-2.7.1/Object.html#method-i-tap):

```
create(:project, name: project_name).tap do |project|
  create(:relationship, project: project, user: user, role: Role.hashed['maintainer'])
end
```

Reply:
@dmarcoux thats nice, thanks, haven't seen the ```tap``` yet :)

Reply:
@dmarcoux applied